service/worker/batcher/activities.go

```diff
@@ -244,7 +244,16 @@ func startTaskProcessor(
 			case BatchTypeSignal:
 				err = processTask(ctx, limiter, task,
 					func(workflowID, runID string) error {
-						return sdkClient.SignalWorkflow(ctx, workflowID, runID, batchParams.SignalParams.SignalName, batchParams.SignalParams.Input)
```

Comment:
The data converter is no-op for input of type []byte, that is what it used to be. Looks like we have changed that behavior?